### PR TITLE
Adding files required for app-sre pipeline

### PR DESCRIPTION
**Pull request by blrm**
This adds the required templates and scripts for the managed-velero-operator to be deployed via the app-sre pipeline.

### CODE REVIEW

File: config/templates/managed-velero-operator-csv-template.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+metadata:
+  name: managed-velero-operator-0.0.1
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: "false"
+    description: Operator to manage installation of Velero for OpenShift Clusters.
+    containerImage: quay.io/app-sre/managed-velero-operator:latest
+    createdAt: "2019-04-10T17:34:33Z"
+    support: Christoph Blecker
```
[the remark this answers is not in the document]
Comment (blrm) [reply config/templates/managed-velero-operator-csv-template.yaml]: ok, this has been corrected.

File: config/templates/managed-velero-operator-csv-template.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    strategy: deployment
+    spec:
+      clusterPermissions:
+      - serviceAccountName: managed-velero-operator
+        # Rules will be added here by the generate-operator-bundle.py script.
+      deployments:
+      - name: managed-velero-operator
+        # Deployment spec will be added here by the generate-operator-bundle.py script.
+  customresourcedefinitions:
+    owned:
```
[the remark this answers is not in the document]
Comment (blrm) [reply config/templates/managed-velero-operator-csv-template.yaml]: The latest commit adds the additional manifests to the CSV and the olm-artifact template.

**Comment by cblecker** [comment]
/lgtm
/approve

This seems sane to me, and needs to merge before the jenkins pipelines can even be attempted.

Let's see how this goes..

**Pull request merged**